src/lib/wicked/adapter.js

```diff
@@ -113,6 +113,35 @@ class WickedAdapter {
         return result;
     }
 
+
+    /*
+     * Obtains the DNS global settings from /etc/sysconfig/network/config
+     */
+    async dnsSettings() {
+        const filePath = `/etc/sysconfig/network/config`;
+        const file = await new SysconfigFile(filePath).read();
+        const policy = await file.get("NETCONFIG_DNS_POLICY") || "";
```

Comment:
Ah! you have the default value for those properties already here!


BTW, if I'm not wrong, you don't need to use `await` with the `get` method.